manifest.toml

```diff
@@ -53,6 +53,17 @@ ram.runtime = "50M"
 
     [resources.ports]
 
+    [resources.sources]
+
+        # Source is only added for autoupdate, it's not actually being downloaded
+        # For updating version n8n version needs to be updated in `_common.sh`
```

Comment:
We can avoid the need for updating `_common.sh` by replacing all instances of `$n8n_version` by `$(ynh_app_upstream_version)`.

Reply:
That's a great suggestion. Thanks @tituspijean . I replaced hard-coded version in `_common.sh` with `$(ynh_app_upstream_version)`, see https://github.com/YunoHost-Apps/n8n_ynh/pull/56/commits/5ad5892b88c7727d8a3de08400742a3b7869b12e . Is that good with you?